docs/cli.md

```diff
@@ -2,9 +2,11 @@
 
 DataHub comes with a friendly cli called `datahub` that allows you to perform a lot of common operations using just the command line. 
 
-## Install
+## Use through docker image
 
-### Using pip
+You can use the `datahub-ingestion` docker image as explained in [Docker Images](../docker/README.md). In case you are using Kubernetes you can start a pod with the `datahub-ingestion` docker image, log onto a shell on the pod and you should have the access to datahub CLI in your kubernetes cluster.
+
+## Use through installing python package through pip
```

Comment:
Maybe we can say "Install using Pip"

Reply:
For the cli our first suggestion should be `using pip` since this is generally a developer laptop install. So I'd prefer to leave the previous organization

## Install

### Using pip
- pip instructions unmodified

### Using docker
- add docker instructions

Reply:
will do